topics/RDMA.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
+Valkey module that can be dynamically loaded on demand.
+
+## Getting Started
+
+RDMA enables direct data exchange between networked computers' main memory,
+bypassing processors and operating systems.
```

Comment:
Can we add more of an introduce to what RDMA is here? This document heavily assumes you know what RDMA is, I would like our documentation to include a better explanation if someone were to find this and is curious about it. Trying to act as someone who doesn't know, I imagine questions like, "How does Valkey use RDMA?" "What type of support do I need for it?" "Is it encrypted?" "How does it work" etc. Worst case, maybe just include a link up here about where to learn more.

Reply:
Because I defined the specification and implemented the `Valkey Over RDMA`, I unconsciously believe that others also understand this. Thanks for pointing out these, `How does Valkey Over RDMA work` and `Is Valkey Over RDMA encrypted` will be added in the next version.